frappe/core/doctype/system_settings/system_settings.js

```diff
@@ -13,3 +13,18 @@ frappe.ui.form.on("System Settings", "refresh", function(frm) {
 	});
 });
 
+frappe.ui.form.on("System Settings", "validate", function(frm) {
+    if(frm.doc.enable_password_policy == 1 && frm.doc.minimum_password_score == ""){
+        validated = false;
+        msgprint("Please select Minimum Password Score");
```

Comment:
All user facing strings must be wrapped with `__("")` in js and `_("")` in python so that they become translatable. This is a minimum requirement